src/Service/ReviewerFilter.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Service;
+
+use App\Api\Issue\IssueApi;
+use App\Model\Repository;
+
+/**
+ * From a large list of possible reviewers, reduce it to just one.
+ *
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+class ReviewerFilter
+{
+    private $issueApi;
+
+    // These users will never be pinged for a review
+    private $blocked = [
+            'fabpot', 'tobion', 'nicolas-grekas', 'stof', 'dunglas', 'jakzal',
+            'xabbuh', 'javiereguiluz', 'lyrixx', 'weaverryan', 'chalasr', 'ogizanagi',
+            'sroze', 'yceruto', 'nyholm', 'wouterj', 'derrabus', 'jderusse',
```

Comment:
does the bot have access to the mergers team?

Reply:
Im not sure. 

But that would be a great future update to implement.